pkg/components/modelserver/modelserver.go

```diff
@@ -154,10 +144,17 @@ func NewDeployment(k *v1alpha1.Kepler) *appsv1.Deployment {
 	}
 }
 
-func NewService(k *v1alpha1.Kepler) *corev1.Service {
-	exporterPort := int32(k.Spec.Exporter.Port)
+func NewService(ms *v1alpha1.ModelServerSpec) *corev1.Service {
+	port := defaultPort
+	if ms.Port > 0 {
+		port = ms.Port
+	}
```

Comment:
this shouldn't be needed since the api server should reject anything less than 1

Reply:
I don't see webhook that check Kepler CR?
I need this at least for the unit test because it is not filled by kbuilder as in csv file. 
Even though, in deployment, if user does not define it will be 8100, I just have this in case to make sure we set the default port if it is not properly defined.

Reply:
We don't need webhook if we add the right markers 

```
// +kubebuilder:validation:Minimum
specifies the minimum numeric value that this field can have. Negative numbers are supported.
```
https://book.kubebuilder.io/reference/markers/crd-validation.html 
see: exporter.deployment.port

This also reminds me that we should also separate out `deployment` related settings to a `DeploymentSpec`